controllers/rhmi/rhmi_controller.go

```diff
@@ -82,7 +82,7 @@ import (
 const (
 	deletionFinalizer                = "configmaps/finalizer"
 	previousDeletionFinalizer        = "finalizer/configmaps"
-	DefaultInstallationName          = "rhmi"
+	DefaultInstallationName          = "rhoam"
```

Comment:
if rhoam is the default, why are specifying rhoam again in the line below? I think we should agree that rhoam is the default type without specifying it again

Reply:
Did some looking into this. Turns out both constants here are only used in the one non-export helper function which would return one or the other. Have removed the helper function and both constants.